include/forkserver.h

```diff
@@ -30,6 +30,7 @@
 
 #include <stdio.h>
 #include <stdbool.h>
+#include "types.h"
```

Comment:
types.h was already included in the next line :-)

Reply:
great find :)